package.json

```diff
@@ -12,7 +12,9 @@
     "devDependencies": {
         "chai": "^4.1.2",
         "chai-as-promised": "^7.1.1",
+        "dotenv": "^6.0.0",
         "electron": "1.8.4",
+        "electron-context-menu": "^0.10.0",
```

Comment:
What are these two new dependencies for?

Reply:
The `electron-context-menu` module gives the `inspect element` right-click functionality, in the electron app, only in `development` mode. 

It helped me with debugging. I could take it out if you're uncomfortable with it.

Reply:
I see. I will test it when I test your branch, but if it's just a development dependency it should be fine.